src/components/toolbox/pageHeader/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './pageHeader.css';
+
+const PageHeader = ({
+  title, subtitle, children,
+}) => (
+  <header className={styles.wrapper}>
+    <span>
+      <h1>{title}</h1>
+      <div className={styles.subtitle}>{subtitle}</div>
+    </span>
```

Comment:
hey Vit I was checking this and I know that works, and it is not a big deal, will not be better to wrap the title and subtitle in the div tag and then the subtitle to be a span?

Reply:
What I was trying to express is that there are multiple `span`s in a row, but `div` has its own row in its container.  But I don't have a strong opinion about this. So I changed it to your suggestion in https://github.com/LiskHQ/lisk-hub/pull/2225/commits/a0e2be96cf87f1e8f509a18b4f00ef739b6bd733